src/agent/main.cpp

```diff
@@ -145,11 +149,12 @@ static int realmain(int argc, char *argv[])
     const char *              backboneInterfaceName = "";
     bool                      verbose               = false;
     bool                      printRadioVersion     = false;
+    bool                      enableAutoAttach      = true;
     std::vector<const char *> radioUrls;
 
     std::set_new_handler(OnAllocateFailed);
 
-    while ((opt = getopt_long(argc, argv, "B:d:hI:Vv", kOptions, nullptr)) != -1)
+    while ((opt = getopt_long(argc, argv, "B:d:hI:Vva:", kOptions, nullptr)) != -1)
```

Comment:
```suggestion
    while ((opt = getopt_long(argc, argv, ":B:d:hI:Vva:", kOptions, nullptr)) != -1)
```
Make `--enable-auto-attach` optional? https://pubs.opengroup.org/onlinepubs/9699919799/functions/getopt.html

Reply:
It is optional. Defaults to `true`. It looks like adding a colon doesn't change the behavior.

Reply:
it allows optional argument.

Reply:
It is already optional without the colon. You can see the test script that checks the default behavior.